tests/models/test_prefix_graph.py

```diff
@@ -37,24 +39,30 @@ def get_pandas_record_set(tmpdir):
 
 
 @lru_cache(maxsize=None)
-def get_windows_conda_build_record_set(tmpdir):
+def get_windows_conda_build_record_set(tmpdir, merge_noarch=True):
```

Comment:
Ideally we can make the tests that depend on this helper with `merge_noarch=False`, but for some reason some tests require a "superchannel" with merged subdirs to pass with libmamba. It seems to be, once again, an artifact of how we are exporting the mocked channels to JSON files so libmamba can load them.